changelog/6845.improvement.md

```diff
@@ -0,0 +1,2 @@
+Check action fingerprints in rules to make sure actions always set slots
```

Comment:
What is an "action fingerprint"? I think we don't need a changelog for this as the rule policy in itself wasn't released yet.

Reply:
`action fingerprint` is smth already exists in rasa oss:
https://github.com/RasaHQ/rasa/blob/91a5b5f7ebcc50590beb0de5547b856cdd3992c2/rasa/core/policies/ensemble.py#L213

Reply:
I'll remove changelog